ui/app/pages/keychains/index.scss

```diff
@@ -93,8 +93,7 @@
 }
 
 .import-account__faq-link {
```

Comment:
This class appears to be unused

Reply:
The same goes for:
* `import-account__dropdown`
* `import-account__description-text`
* `import-account__input-error-message`
* `import-account__file-input-label`
* `import-account__file-name`